Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 
 gem 'cocoapods'
 gem 'cocoapods-update-if-you-dare', '~> 0.1.1'
+gem 'colored'
```

Comment:
But if it's not released soon, the PR looks cool and we could merge it as is. Does it sound fine for you @garriguv ?